pkg/helpers/identifier_string_parser.go

```diff
@@ -4,17 +4,16 @@ import (
 	"encoding/csv"
 	"fmt"
 	"strings"
+
```

Comment:
I wonder where this whole parsing logic should reside, and I think that:
- parsing fits best in the SDK package (because it's SF logic, and nothing tf-specific)
- encoding should reside in tf specific package, because it it where we will use it (and this is nothing to do with the SDK)

Reply:
Yeah, I forgot to add this to the pr notes, but I just wasn't sure where this should go. Moved to appropriate packages.